scripts/javadoc/preserve_olddocument.sh

```diff
@@ -0,0 +1,37 @@
+echo "release_version=$release_version"
+previousversion=$(grep -o "AWS SDK for Android - [0-9]\+.[0-9]\+.[0-9]\+" docs/reference/allclasses-frame.html | grep -o "[0-9]\+.[0-9]\+.[0-9]\+")
```

Comment:
"AWS SDK for Android" -> "Amplify Android"

Reply:
```suggestion
previousversion=$(grep -o "Amplify Android - [0-9]\+.[0-9]\+.[0-9]\+" docs/reference/allclasses-frame.html | grep -o "[0-9]\+.[0-9]\+.[0-9]\+")
```